src/webutil/tableState.js

```diff
@@ -0,0 +1,185 @@
+// @flow
+
+import {type PluckFn} from "../util/sortBy";
+import sortBy from "../util/sortBy";
+import cloneDeep from "lodash.clonedeep";
+import {useMemo, useState} from "react";
+
+export const SortOrders = {
+  ASC: "asc",
+  DESC: "desc",
+};
+type SortOrderTypes = "asc" | "desc";
```

Comment:
I did find that in my research. The reason I did it this way was because the TableSortLabel can use the lowercase strings as input, which makes this line cleaner:
```
direction={
                    ts.sortName === DATE_SORT[0] ? ts.sortOrder : SortOrders.ASC
                  }
```

I didn't check if it would accept uppercase though. I'll play with it to see.

Reply:
Yeah as I guessed, the TableSortLabel wants it lowercase. So we can make the constant names lowercase to add this sugar or we can leave it. I'll probably leave it.

Reply:
For another take on how we could set up options like this, take a look at the Direction const in core/graph:

https://github.com/sourcecred/sourcecred/blob/651db6de017cc74dfc85f16960144d0849fab105/src/core/graph.js#L151-L160

It sounds like in this case we can't take that approach due to the lowercase keys being expected by TableSortLabel. However, I do recommend using deepFreeze -- allowing something to modify these values at runtime would never be a good thing.